tools/dreport.d/ibm.d/gendumpheader

```diff
@@ -74,10 +74,18 @@ function dump_size () {
 #Function to set dump id to 8 bytes format
 function get_dump_id () {
     # shellcheck disable=SC2154
+    if [ -z $dump_id ]; then
+        dump_id="00000000"
```

Comment:
NO, need of this as we initialize dump id as "0..0"
https://github.com/ibm-openbmc/phosphor-debug-collector/blob/1020/tools/dreport.d/dreport#L61

Reply:
Agree. Will be removing in the next commit.